subsys/net/l2/canbus/canbus.c

```diff
@@ -1077,6 +1148,329 @@ static enum net_verdict process_frame(struct net_pkt *pkt)
 	return ret;
 }
 
+#if defined(CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR)
+static void forward_eth_frame(struct net_pkt *pkt, struct net_if *canbus_iface)
+{
+	pkt->iface = canbus_iface;
+	net_if_queue_tx(canbus_iface, pkt);
+}
+
+static struct net_ipv6_hdr *get_ip_hdr_from_eth_frame(struct net_pkt *pkt)
+{
+	return (struct net_ipv6_hdr *)((u8_t *)net_pkt_data(pkt) +
+		sizeof(struct net_eth_hdr));
+}
+
+enum net_verdict net_canbus_translate_eth_frame(struct net_if *iface,
+						struct net_pkt *pkt)
+{
+	struct net_linkaddr *lladdr = net_pkt_lladdr_dst(pkt);
+	struct net_pkt *clone_pkt;
+	struct net_if *canbus_iface;
+
+	/*Forward only IPv6 frames*/
```

Comment:
Could you add spaces here (and also the comments in other parts of this PR). So `/* xxx */` instead of `/*xxx*/`